### DISCUSSION TITLE
Change metadata generator

### DISCUSSION THREAD

**Discussion by azxqw** [Security-Onion-Solutions/securityonion#3608] [Unsupported Versions]
Hi,
Is it possible to change the metadata generator from suricata to zeek and vice versa after the installation and setup are done?

**Comment by palichx** [comment]
Hi,
"You can change mdengine in the global pillar to ZEEK instead of SURICATA."
https://github.com/Security-Onion-Solutions/securityonion/discussions/3259#discussioncomment-434499
